pkg/cognito/authcertdeleter.go

```diff
@@ -22,6 +22,10 @@ func (c *AuthCertDeleter) DeleteAuthCert(domain string) error {
 		return err
 	}
 
+	if domain == "" {
```

Comment:
I think we're supposed to put things like this into a struct as early as possible and find problems like this with struct.Validate()

Reply:
Yes, that would be better - think i need a new round of cleanup later, want this in for now